.gitignore

```diff
@@ -1,3 +1,4 @@
 *.py[co]
 .env
 /static
+*.sqlite3
```

Comment:
Since we are not using this in the project, please remove it from here. You can always ignore this locally.